spec/models/spotlight/dor/indexer_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 
+# rubocop: disable Metrics/BlockLength
```

Comment:
What would you think about disabling this for the `spec` directory entirely?

Reply:
+1 that notion ☝️

Reply:
Fine with me. The rubocop todo has a huge number for this cop already.